sdks/java/extensions/euphoria/src/main/java/org/apache/beam/sdk/extensions/euphoria/core/client/type/TypeAwareness.java

```diff
@@ -20,7 +20,7 @@
 import java.util.Optional;
 import org.apache.beam.sdk.values.TypeDescriptor;
 
-public class TypeAwares {
+public class TypeAwarnes {
```

Comment:
also a typo, it should be TypeAwareness and I think originally it was TypeAware

Reply:
You are right.